components/KiboOrderItemDetails.vue

```diff
@@ -1,29 +1,37 @@
 <template>
   <div>
     <slot name="header-action" />
-    <div>
-      <hr class="details-hr" />
-    </div>
     <div class="order">
       <div class="order-header-container order__item-box">
         <div class="order-header">
-          <SfProperty :name="$t('Your Order:')" :value="orderNumber" />
+          <SfProperty :name="$t('Order Number')" :value="orderNumber" />
           <SfProperty
-            :name="$t('Order Date:')"
+            :name="$t('Order Date')"
             :value="orderSubmittedDate"
             class="sf-order-summary__property"
           />
-          <SfProperty
-            :name="$t('Order Total:')"
-            :value="$n(orderTotal, 'currency')"
-            class="sf-order-summary__property"
-          />
+          <SfProperty :name="$t('Order Total')" class="sf-order-summary__property">
+            <template #value>
+              <span class="sf-property__value">
+                {{ $n(orderTotal, "currency") }}
+                <span class="items">{{
+                  `(${orderItemCount} item${orderItemCount > 1 ? $t("s") : $t("")})`
```

Comment:
follow this https://kazupon.github.io/vue-i18n/guide/pluralization.html#accessing-the-number-via-the-pre-defined-argument

Reply:
Resolved